### DISCUSSION TITLE
Whether slint can be used in Arduino

### DISCUSSION THREAD

**Discussion by modi12jin** [slint-ui/slint#4103] [Q&A]
Since slint supports C++, is it possible to use it in arduino? If so, can you provide a migration guide or demo video

**Comment by hunger** [comment]
The arduino seems to have rust support, so I see no reason why it should not be possible to make Slint work on it. You need rust since Slint itself is a library written in Rust.

I would love seeing a video, too, or a PR adding isntructions on how to get up and running or arduino boards.
